style.css

```diff
@@ -0,0 +1,250 @@
+.wrapper {
+  display: grid;
+  grid-template-columns: 33% 33% 33%;
+  grid-gap: 10px;
+  background-color: #fff;
+  color: #444;
+}
+
+.box {
+  height: auto;
+  width: auto;
+}
+
+.box img {
+  width: 100%;
+  height: auto;
+}
+
+.image-size {
+  width: 20px;
+  height: 20px;
+}
+
+.btn-size {
+  width: 20px;
+  height: 20px;
+}
+
+.btn-color {
+  background-color: #6288a5;
+  color: white;
+  padding: 10px 10px 10px 10px;
+}
+
+.nav-bar {
+  width: 100%;
+  -moz-box-shadow: 0 8px 6px -6px #999;
+  box-shadow: 0 8px 6px -6px #999;
```

Comment:
The navbar should be styled with a fixed position

Reply:
hey man thanks for the code review, we are having a problem with the 7th one because in our link to the page there isnt the aside section https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0 this is the link in our path wright

Reply:
Hi guys, it does show for me with that link. Is it happening for both of you?